sale_order_price_recalculation/tests/test_sale_order_price_recalculation.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2015 Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+# Copyright 2016 Vicent Cubells <vicent.cubells@tecnativa.com>
+# Copyright 2017 David Vidal <david.vidal@tecnativa.com>
+# Copyright 2018 Duc Dao Dong <duc.dd@komit-consulting.com>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+from odoo.tests import common
+
+
+class TestSaleOrderPriceRecalculation(common.TransactionCase):
```

Comment:
Please:
1. use SavepointCase, seems pointless to re-create these records for each test
2. disable tracking globally using `cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))`